files/image_config/caclmgrd/caclmgrd

```diff
@@ -88,9 +90,17 @@ class ControlPlaneAclManager(object):
     }
 
     def __init__(self):
-        # Open a handle to the Config database
-        self.config_db = ConfigDBConnector()
-        self.config_db.connect()
+        SonicDBConfig.load_sonic_global_db_config()
+        self.config_db_map = {}
+        self.iptables_cmd_prefix = {}
```

Comment:
Wondering if this variable name should also mention something about "namespace" or "ns". Like `iptables_cmd_ns_prefix`?

Reply:
@jleveque yes , that is more specific. Updated.